plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -4095,3 +4095,37 @@ function wc_set_hooked_blocks_version_on_theme_switch( $old_name, $old_theme ) {
 		add_option( $option_name, WC()->version );
 	}
 }
+
+/**
+ * Add aria-label to pagination numbers.
+ *
+ * @param string $r    HTML output.
+ * @param array  $args An array of arguments. See paginate_links()
+ *                     for information on accepted arguments.
+ *
+ * @return string
+ */
+function wc_add_aria_label_to_pagination_numbers( $r, $args ) {
```

Comment:
Can we rename the `$r` variable into more meaningful name? Like `$html` or something? Unless there's a good reason to keep it `$r`.

```suggestion
function wc_add_aria_label_to_pagination_numbers( $html, $args ) {
```

Reply:
@kmanijak I used `$r` because that's how the filter is documented on https://developer.wordpress.org/reference/functions/paginate_links/, but I agree that `$html` provides more meaningful name and will change it. Thanks!

Reply:
✅ Done
[Rename $r parameter to $html](https://github.com/woocommerce/woocommerce/pull/50447/commits/ed695c3a92ce6cb86af52d66aed8aca15f756a9e)